pontoon/base/utils.py

```diff
@@ -10,7 +10,7 @@
 import zipfile
 
 from collections import defaultdict
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
```

Comment:
We should be able to use `timezone` from `django.utils` everywhere I believe. So you could drop this import and also the `dj_timezone` alias below.

Reply:
Oh, I see now that `django.utils.timezone.utc` is deprecated.